<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainpage</class>
 <widget class="QMainWindow" name="mainpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>480</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>1000</height>
   </size>
  </property>
  <property name="font">
   <font>
    <stylestrategy>PreferDefault</stylestrategy>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(11, 11, 11)
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>601</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:url(:/img/bckgg.jpg)</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="logoutButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: grey;
color: white;
border-style:outset;
border-radius:10px;
border-color:grey;
}
QPushButton:hover{
background-color: rgb(158, 6, 47);
color: white;
border-style:outset;
border-radius:10px;
border-color:grey;
}</string>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="searchBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>281</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(252, 252, 126);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchButtomn">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>23</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="addReteta">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>20</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: grey;
color: white;
border-style:outset;
border-radius:10px;
border-color:grey;
}
QPushButton:hover
{
background-color: rgb(187, 171, 49);
color: white;
border-style:outset;
border-radius:10px;
border-color:grey;
}
</string>
      </property>
      <property name="text">
       <string>Adauga Reteta</string>
      </property>
     </widget>
     <widget class="QPushButton" name="favoritesButton">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>20</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/favorites.png)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QListWidget" name="listaCautari">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>941</width>
        <height>511</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 170);</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="searchBox_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>43</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 127);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchButtonNume">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>43</y>
        <width>23</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="top">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">selection-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>TOP 5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: grey;
color: white;
border-style:outset;
border-radius:10px;
border-color:grey;
}
QPushButton:hover
{
background-color: rgb(187, 171, 49);
color: white;
border-style:outset;
border-radius:10px;
border-color:grey;
}</string>
      </property>
      <property name="text">
       <string>Retetele mele</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
